phpunit.xml.dist

```diff
@@ -1,11 +1,20 @@
-<phpunit
-	bootstrap="vendor/autoload.php"
-	colors="true"
-	>
-	<testsuites>
-		<testsuite>
-			<directory prefix="spec-" suffix=".php">tests/</directory>
-			<directory prefix="test-" suffix=".php">tests/</directory>
-		</testsuite>
-	</testsuites>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/4.8/phpunit.xsd"
+		 bootstrap="vendor/autoload.php"
+		 backupGlobals="false"
+		 beStrictAboutCoversAnnotation="true"
+		 beStrictAboutOutputDuringTests="true"
+		 beStrictAboutTestsThatDoNotTestAnything="true"
+		 beStrictAboutTodoAnnotatedTests="true"
+		 colors="true"
+		 verbose="true">
+	<testsuite name="wp-cli/package-command tests">
```

Comment:
```suggestion
	<testsuite name="default">
```
It can remain default given this test file is specific to this package. Also, I think we should add `defaultTestSuite="default"` attribute in the `<phpunit>` element?

Reply:
This is more consistent with other commands though. Main thing is that the `name` is defined, which it previously wasn't.